contributions/catalog/controllers/config.py

```diff
@@ -1,43 +1,24 @@
 import os
+from dotenv import load_dotenv
 
 
+# Load .env file
+load_dotenv()
+
 class Config(object):
-    CONTRIBUTION_BUILDING_BLOCK_URL = os.getenv("CONTRIBUTION_BUILDING_BLOCK_URL",
-                                                "http://localhost:5000/contributions")
+    OAUTHLIB_INSECURE_TRANSPORT = True
+    CONTRIBUTION_BUILDING_BLOCK_URL = os.getenv("CONTRIBUTION_BUILDING_BLOCK_URL", "http://localhost:5000/contributions")
     MONGO_URL = os.getenv('MONGO_URL', 'mongodb://localhost:27017')
     DB_NAME = os.getenv("MONGO_DATABASE", "contribution")
     DB_COLLECTION = os.getenv("MONGO_DATABASE", "catalog")
-
     URL_PREFIX = os.getenv("URL_PREFIX", "")
-
     DBTYPE = 'mongoDB'
     SECRET_KEY = os.getenv("SECRET_KEY", "SECRET_KEY")
     AUTHENTICATION_TOKEN = os.getenv("AUTHENTICATION_TOKEN", "...")
     ROKWIRE_API_KEY = os.getenv("ROKWIRE_API_KEY", "...")
     DEBUG = True
-
     client_id = os.getenv("CLIENT_ID", "NO ID")
     client_secret = os.getenv("CLIENT_SECRET", "NO SECRET")
-    authorization_base_url = 'https://github.com/login/oauth/authorize'
-    token_url = 'https://github.com/login/oauth/access_token'
+    authorization_base_url = os.getenv("authorization_base_url", 'https://github.com/login/oauth/authorize')
```

Comment:
For consistency, please capitalize the config and env variable names below. You'll also have to update the places where these are being referred.

```
    client_id = os.getenv("CLIENT_ID", "NO ID")
    client_secret = os.getenv("CLIENT_SECRET", "NO SECRET")
    authorization_base_url = os.getenv("authorization_base_url", 'https://github.com/login/oauth/authorize')
    token_url = os.getenv("token_url", 'https://github.com/login/oauth/access_token')
```

Reply:
Updated